docs/user-guide/app_deletion.md

```diff
@@ -0,0 +1,26 @@
+# App Deletion
+
+## Cascade App Deletion
+
+Apps can be deleted with or without a cascade option. Cascade deletion deletes the both app's and its resources, rather than only the app. 
+
+
+```bash
+argocd app delete APPNAME --cascade
+```
+
+For the technical amongst you, this adds the following Kubernetes finalizer to the app's manifest: 
+
+
+```yaml
+metadata:
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+```
+
+Argo CD's app controller notices this and will then delete both the app and its resources.
+
+!!! warning
```

Comment:
It sounds like controller deletes app with resources as long as finalizer is added and it is impossible to Cascade delete with kubectl. This is not correct.

If finalizer is added then 'kubectl delete' triggers cascade delete

Reply:
Good point. I’ll update.

Reply:
I've updated now. Can I please request a re-review.